Microsoft.Research/RegressionTest/ClousotTests/AsyncTestDriver.cs

```diff
@@ -20,147 +9,146 @@
 
 namespace Tests
 {
-  public class AsyncTestDriver
-  {
-    delegate void IsolatedAction<in T>(T obj, out Exception exceptionThrown, out string dataReceived);
-
-    public static readonly uint MaxWaitHandles_Default = Math.Max(1, Math.Min(4, (uint)(Environment.ProcessorCount - 1)));
-    public static readonly uint MaxWaitHandles_AllButOne = Math.Max(1, (uint)(Environment.ProcessorCount - 1));
+    public class AsyncTestDriver
+    {
+        private delegate void IsolatedAction<in T>(T obj, out Exception exceptionThrown, out string dataReceived);
 
-    private static readonly int SingleTestMaxWait = 200000;
+        public static readonly uint MaxWaitHandles_Default = Math.Max(1, Math.Min(4, (uint)(Environment.ProcessorCount - 1)));
+        public static readonly uint MaxWaitHandles_AllButOne = Math.Max(1, (uint)(Environment.ProcessorCount - 1));
 
-    private readonly Action<Options, Output> action;
-    private readonly IsolatedAction<Options> actionDelegate;
-    private readonly Func<Options, bool> skipTest;
-    private Dictionary<string, IAsyncResult> testAsyncResults;
-    private readonly uint maxWaitHandles;
-    private WaitHandle[] waitHandles;
-    private int nbWaitHandles;
-    private bool beginTestsProcessed = false;
-    private bool orderReversed = false;
+        private static readonly int SingleTestMaxWait = 200000;
 
-    public string BeginMessage;
+        private readonly Action<Options, Output> action;
+        private readonly IsolatedAction<Options> actionDelegate;
+        private readonly Func<Options, bool> skipTest;
+        private Dictionary<string, IAsyncResult> testAsyncResults;
+        private readonly uint maxWaitHandles;
+        private WaitHandle[] waitHandles;
+        private int nbWaitHandles;
+        private bool beginTestsProcessed = false;
+        private bool orderReversed = false;
 
-    public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest)
-      : this(action, skipTest, MaxWaitHandles_Default)
-    { }
+        public string BeginMessage;
 
-    public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest, uint maxWaitHandles)
-    {
-      this.action = action;
-      this.actionDelegate = this.ActionAsIsolated;
-      this.skipTest = skipTest;
-      this.maxWaitHandles = maxWaitHandles;
-    }
+        public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest)
+          : this(action, skipTest, MaxWaitHandles_Default)
+        { }
 
-    // We have no control on the order of the tests, so we make sure
-    // to always call Begin before End
-
-    public void BeginTest(Options options)
-    {
-      if (this.skipTest(options))
-        return;
+        public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest, uint maxWaitHandles)
+        {
+            this.action = action;
```

Comment:
I explicitly disabled that - I left a message in Gitter explaining why (basically there is so much code to reformat that we need to do the safe option first and then we can consider renaming things in the future).

I closed all the other formatting PRs so I can update them to also *not* rename fields.